backend/app/models/log_records.py

```diff
@@ -14,7 +14,7 @@ class LogRecords(db.Model):
     # TODO: replace open String fields with VarChar(NUM_CHARS)
     note = db.Column(db.String, nullable=False)
     tags = db.Column(db.ARRAY(db.String), nullable=True)
-    building = db.Column(db.String, nullable=False)
+    building_id = db.Column(db.Integer, db.ForeignKey("buildings.id"), nullable=False)
 
```

Comment:
going off the above comment, you can add something like this (keeping the `building_id` column):`
```
building = db.relationship('Buildings', back_populates='log_record')
```

when we query a log record, we can easily access the building through this property